boards/arm/frdm_kw41z/frdm_kw41z.dts

```diff
@@ -79,3 +79,32 @@
 	status = "ok";
 	current-speed = <115200>;
 };
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * http://docs.zephyrproject.org/devices/dts/flash_partitions.html
+	 */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+		
```

Comment:
while you are at it, can you please add a boot_partition?

Reply:
I'm not sure how to test something like that? How big would I make it? 

The KW41Z never seemed to be built with a boot partition but the BLE stuff needed a file storage partition so I just added what was needed for the BLE stuff.